sentences/es/cover_HassOpenCover.yaml

```diff
@@ -3,14 +3,14 @@ intents:
   HassOpenCover:
     data:
       - sentences:
-          - "abre <name>"
-          - "abre <name> <area>"
+          - "(abre|abrir|sube|subir) <name>"
```

Comment:
Consider moving this to an expansion rule

Reply:
Done